cypress/integration/home.spec.js

```diff
@@ -0,0 +1,69 @@
+const headerItemText = [
+  "API CATALOGUE",
+  "SIGN IN",
+];
+
+const sidebarItemText = [
+    "Mission",
+    "The need of a developer hub",
+    "API Specifications",
+    "Our ways of working",
+];
+
+const footerItemText = [
+  "Hackney sites",
+  "Accessibility",
+  "Contact us",
+];
+
+function elementContainsTextArray(element, textArray) {
+  element.each((item, index) => {
+    expect(item.text()).to.eq(textArray[index]);
+  });
+}
+
+describe("Home page integration test", () => {
+  beforeEach(() => {
+    cy.visit("/")
+  });
+
+  it("Navbar test", () => {
```

Comment:
The header & footer aren't specific to the home page so the tests for them can probably go into another file :)

Reply:
I am planning to use this for Makers if that's ok